component/src/main/java/com/synopsys/integration/alert/component/settings/proxy/database/accessor/SettingsProxyConfigAccessor.java

```diff
@@ -108,10 +105,11 @@ public SettingsProxyModel updateConfiguration(UUID configurationId, SettingsProx
 
     @Override
     @Transactional(propagation = Propagation.REQUIRED)
-    public void deleteConfiguration(UUID configurationId) {
-        if (null != configurationId) {
-            settingsProxyConfigurationRepository.deleteById(configurationId);
-        }
+    public void deleteConfiguration(String configurationName) {
+        settingsProxyConfigurationRepository.findByName(configurationName)
+            .stream()
+            .findFirst()
+            .ifPresent(settingsProxyConfigurationEntity -> settingsProxyConfigurationRepository.deleteById(settingsProxyConfigurationEntity.getConfigurationId()));
```

Comment:
Couldn't you just create a method in `settingsProxyConfigurationRepository ` to delete by name?